src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -74,6 +75,10 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
 
   const scheduledStartDate = sovereignVenue?.start_utc_seconds;
 
+  const [filterRelatedEvents, setFilterRelatedEvents] = useState(
+    venue.id !== sovereignVenue?.id
```

Comment:
it'd be updated via 

> const isNotSovereignVenue = venue.id !== sovereignVenue?.id;

Reply:
looks like there was a bug, so I fixed that setting it to `false`
https://github.com/sparkletown/sparkle/pull/2352/commits/90366891c725a5ed617297b35a2917d624ca566a